src/commands/upload_file.ts

```diff
@@ -227,6 +226,7 @@ new CLICommand({
 
 			const conflictResolution = parameters.getFileNameConflictResolution();
 			const shouldBundle = !!parameters.getParameterValue(ShouldBundleParameter);
+			const remoteFilePath = parameters.getParameterValue(RemotePathParameter);
```

Comment:
nit: Could improve the locality of this variable by declaring it about the if check